pmg/views.py

```diff
@@ -234,10 +235,16 @@ def bill(bill_id):
         'introduced': 1,
     }
     history = bill_history(bill)
+
+    if bill['status']:
```

Comment:
to fix a key error for `bill['status']` do `if 'status' in bill` - this is still a key error and it's only testing whether `bill['status']` is falsy